app/src/main/res/values/colors.xml

```diff
@@ -1,23 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="primary">#FBC812</color>
```

Comment:
Thank you 👍 
But, I think this is not related this PR. And the git log gets dirty. 😭 
Please revert this. and If you can please send other pull requests. 🙇

Reply:
Oh, sorry... I'll be careful from now on. Please just a moment while finish cleaning it.